app/content/drill/[id]/index.js

```diff
@@ -48,7 +49,7 @@ export default function Index() {
             }}
             color={"#F24E1E"}
           />
-          <Appbar.Content title={drillInfo["drillType"]} />
+          <Appbar.Content title={getCombinedDrillTitle(drillInfo)} />
```

Comment:
When testing this out on IOS the "20 shot challenge | 100 - 150 yds" got cut off. I will try to get a screen shot uploaded

Reply:
![image](https://github.com/Golf-Drill-Challenge-App/Golf-App/assets/91503842/a7a95416-c8d3-42e7-bdce-816719a2b1db)

Reply:
You can combine all of these into 1 big review you know...

Reply:
oops, might have messed up with my review then lol. I will be more careful next review I do